src/lib.rs

```diff
@@ -392,16 +438,17 @@ pub struct Post<BlockNumber, Moment, AccountId> {
     /// to a thread when it was added.
     /// Is needed to give light clients assurance about getting all posts in a given range,
     // `created_at` is not sufficient.
-    post_nr: u32,
+    /// Starts at 1 for first post in thread.
+    post_nr_in_thread: u32,
 
-    /// Initial text of post
-    initial_text: Vec<u8>,
+    /// Current text of post
+    current_text: Vec<u8>,
 
     /// Possible moderation of this post
     moderation : Option<ModerationAction<BlockNumber, Moment, AccountId>>,
     
     /// Edits of post ordered chronologically by edit time.
-    post_text_edits: Vec<PostTextEdit<BlockNumber, Moment>>,
+    expired_post_texts: Vec<ExpiredPostText<BlockNumber, Moment>>,
```

Comment:
Is `expired` the best word here? Maybe something like `old_post_texts` or something, just thinking. Because a text is actually **replaced** by the post author.

Reply:
Assuming you are talking about both variable `expired_post_texts` and type `ExpiredPostText`. I think expired is good, because it states very clearly that its no longer valid as the post text, "old" is a bit more ambiguous, it just says it was text a long time ago.

I will change if you give me a specific suggestion you think is better, but this seems very minor.

Reply:
What about this?
struct `PostTextChange` and `text_change_history` as a field name.

```rust
struct Post {
    // ....
    text_change_history: Vec<PostTextChange<BlockNumber, Moment>>,
    // ....
}
```

Reply:
Done.